react/package.json

```diff
@@ -29,19 +29,19 @@
     "@haiilo/catalyst": "workspace:*"
   },
   "peerDependencies": {
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0"
+    "react": "16.9.0",
```

Comment:
peer dependencies should not be pinned. Should probably be something like `>=16.9.0`